mmv1/templates/terraform/examples/cloud_run_service_ingress.tf.erb

```diff
@@ -0,0 +1,27 @@
+# [START cloud_run_service_ingress]
+resource "google_cloud_run_service" "<%= ctx[:primary_resource_id] %>" {
+    name     = "<%= ctx[:vars]['cloudrun_service_name'] %>"
+    location = "us-central1"
+
+    template {
+      spec {
+        containers {
+          image = "gcr.io/cloudrun/hello" #public image for your service
+        }
+      }
+    }
+    traffic {
+      percent         = 100
+      latest_revision = true
+    }
+    metadata {
+      annotations = {
+        # For valid annotation values and descriptions, see
+        # https://cloud.google.com/sdk/gcloud/reference/run/deploy#--ingress
+        "run.googleapis.com/ingress" = "internal"
+
```

Comment:
nit: could we remove the extra whitespace here?

Reply:
Done.